File: benchmarks/var.cpp
#include <iostream>
using namespace std;

#define NUM 10000000
#define DISP 100000

int main(int argc, char* argv[])
{
	int i;
	int tmp;

	for(i = 0; i < NUM; i++)
	{
		tmp = i * 2;

		if(i % DISP == 0)
			cout << "debug: " << i << endl;
	}

	return 0;
}

File: benchmarks/var.pl
#!/usr/bin/perl -w

use constant NUM => 10000000;
use constant DISP => 100000;

for($i = 0; $i < NUM; $i++)
{
	$tmp = $i * 2;
	$tmp++;
#	print "debug: $i\n" if($i % DISP == 0);
}

exit(0);
